ui/apps/platform/src/Containers/NetworkGraph/NetworkGraph.tsx

```diff
@@ -98,7 +98,7 @@ const TopologyComponent = ({
                         <NamespaceSideBar />
                     )}
                     {selectedEntity && selectedEntity?.data?.type === 'DEPLOYMENT' && (
-                        <DeploymentSideBar />
+                        <DeploymentSideBar deploymentId={selectedEntity.id} model={model} />
```

Comment:
why not pass the actual entity data through here instead of passing through the entire data model and the relevant id?

Reply:
I decided to pass the whole model because we need access to various data in this tab:  deployment id, policy ids (from node), and edges from/to deployment. We could pass just what we need, but I thought it would be better to give the model now, and then if we want to refactor and make things more optimal, we can do that separately